internal/constants/constants.go

```diff
@@ -32,7 +34,8 @@ const (
 	SolutionFileBaseName = "solution"
 	InputDirName         = "inputs"
 	OutputDirName        = "outputs"
-	MaxRetries           = 2
+	WorkerStatusIdle     = "idle"
```

Comment:
Good example of constants. This will not change and is the same (constant) for every system.